docs/contributing/index.md

```diff
@@ -0,0 +1,76 @@
+# Contributing
+
+When contributing to this repository, please first discuss the change you wish to make via issue,
```

Comment:
Is it better to ask users to use only issues rather than multiple lines of communication.  While I can see that choice of medium is more inclusive, it makes it more difficult for the community to follow discussion, and to see which issues have already been raised.

Reply:
What do you think? 8a4e2a0

I strongly agree we should not direct users to request support by e-mail.

Reply:
My thought is to direct users to issues